msal/broker.py

```diff
@@ -0,0 +1,198 @@
+"""This module is an adaptor to the underlying broker.
```

Comment:
>This module is an adaptor to the underlying broker. [](http://example.com/codeflow?start=3&length=51)

How about signin API? You should able to call that API with the 0.5 version.

Reply:
This question has been addressed in the internal design doc.